packages/react/label/src/Label.tsx

```diff
@@ -78,12 +71,12 @@ const Label = React.forwardRef((props, forwardedRef) => {
 
   return (
     <LabelContext.Provider value={React.useMemo(() => ({ id, ref: labelRef }), [id])}>
-      <Primitive {...labelProps} as={as} selector={selector} ref={ref} id={id} role="label" />
+      <Primitive {...labelProps} as={as} ref={ref} id={id} role="label" />
```

Comment:
Wonder if it's worth moving `role` before prop spread while we're here?

Reply:
I noticed a bunch of these and was planning to address them in another PR, didn't want them to get lost in a sea of changes.